### PR TITLE
Add genesis to angular guide

### PR DESCRIPTION
**Pull request by patrickoneill-genesis**
Adding a guide for adding genesis to angular.

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
No

Have you checked all new or changed links?
Yes

For reference: 

  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 

**This week's exciting excerpts from the style guide**

- We write in UK English, with Oxford English Dictionary (OED) spellings, grammar and vocabulary.  

Use the present tense wherever possible. Only use another tense where it is strictly necessary.

- Present tense (preferred whenever possible): *The script creates a new folder.*
- Future tense (avoid whenever possible): *The script will create a new folder.*

### CODE REVIEW

File: docs/04_web/04_integrations/02_Angular/06_add_genesis_to_angular.md
Comment (wjhendry) [inline docs/04_web/04_integrations/02_Angular/06_add_genesis_to_angular.md:1]: Last sentence:
Specify your back-end end point.
How do we do that?

Comment (wjhendry) [inline docs/04_web/04_integrations/02_Angular/06_add_genesis_to_angular.md:1]: Line 110 - do we actually have the instructions for this later on? I couldn't find them

Comment (wjhendry) [inline docs/04_web/04_integrations/02_Angular/06_add_genesis_to_angular.md:1]: Line 357:
should this be...
Ensure that the `include` section references the **globals.d.ts** file.

Comment (wjhendry) [reply docs/04_web/04_integrations/02_Angular/06_add_genesis_to_angular.md:1]: format for including is: 
where simple.png is in the folder static/img

Comment (patrickoneill-genesis) [reply docs/04_web/04_integrations/02_Angular/06_add_genesis_to_angular.md:1]: Added

Comment (wjhendry) [reply docs/04_web/04_integrations/02_Angular/06_add_genesis_to_angular.md:1]: this is now clear

**Pull request merged**